app/api/schema/events.py

```diff
@@ -83,7 +83,6 @@ def validate_timezone(self, data, original_data):
     is_billing_info_mandatory = fields.Bool(default=False)
     is_donation_enabled = fields.Bool(default=False)
     is_chat_enabled = fields.Bool(default=False)
-    chat_room_name = fields.Str(allow_none=True)
```

Comment:
If you remove it from schema, how will frontend access it?

Reply:
This should be dumps_only field

Reply:
yes I saw, will push the changes with regex commit